.github/workflows/test-install.yml

```diff
@@ -11,6 +11,9 @@ on:
   schedule:
   - cron: 30 02 * * *      # nightly build
 
+env:
+  FORCE_COLOR: 1
```

Comment:
Why this has to be added in this PR? I just don't see anything related to it. Is it just to ensure that colors are used for visuals? I am asking because I remember that some tests rely on the color and fail when playing with these options but I don't remember the details anymore.

Reply:
This is just an unrelated improvement to add colors to pytest output in GitHub UI. 
I am not aware of any tests that would depend on colors, and we're already using this option in many other workflows (e.g. ci-code) so I think this is a safe change.

Reply:
There was something that failed in #6434 when I enforced colors but I cannot check it because the CI result already removed it. But when the test pass it should be okay!